templates/api/api.go.tpl

```diff
@@ -14,7 +14,7 @@ import (
 	"context"
 )
 
-// Service is the {{ .Config.Name | title }} server interface
+// Service is the {{ regexReplaceAllLiteral "\\W+" .Config.Name " " | title | replace " " "" }} server interface
```

Comment:
Since you use this in a few places, and it might be used in more in the future, you can drop this into https://github.com/getoutreach/stencil-golang/blob/main/templates/_helpers.tpl and reference it here.

Reply:
Excellent, I'll move it to that location.

Reply:
These have been move to the `_helpers.tpl` file.